lektor/publisher.py

```diff
@@ -636,9 +641,15 @@ def git(args, **kwargs):
 
             self.link_artifacts(path)
             self.write_cname(path, target_url)
+
+            default_message = 'Synchronized build'
+            message = extra.get('message')
+            if message is None:
+                message = default_message
+
```

Comment:
I'm reviewing our notes on this, and we found get() method didn't work if there is already a key, here `message` and it was set to `None`.  @arbonap, @gecrooks, and I had tried doing this with the single line method that you also suggested, but found this implementation to be more sound in cases were a message wasn't being passed in.